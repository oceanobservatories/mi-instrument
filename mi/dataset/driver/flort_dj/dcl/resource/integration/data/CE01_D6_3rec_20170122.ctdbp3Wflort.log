2017/01/22 00:28:05.566 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 00:30:03.720 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 00:31:06.294 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 01:28:04.778 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 01:30:03.974 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 01:31:06.256 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 02:28:04.052 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 02:30:03.226 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 02:31:05.483 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 03:28:04.016 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 03:30:04.362 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 03:31:06.616 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 04:28:03.597 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 04:30:12.195 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2017/01/22 04:30:26.806   9.9044,  3.32563,    0.914, 3694, 121, 80, 22 Jan 2017 04:30:20
2017/01/22 04:31:02.001 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 05:28:04.048 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 05:30:03.409 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 05:31:05.669 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 06:28:06.815 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 06:30:02.481 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 06:31:04.747 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 07:28:05.908 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 07:30:03.533 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 07:31:05.799 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 08:28:05.587 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 08:30:03.459 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 08:31:05.724 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 09:28:03.777 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 09:30:03.385 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 09:31:05.654 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 10:28:03.957 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 10:30:03.582 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 10:31:05.832 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 11:28:05.467 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 11:30:03.867 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 11:31:06.125 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 12:28:04.494 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 12:30:02.918 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 12:31:05.491 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 13:28:04.542 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 13:30:02.954 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 13:31:05.208 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 14:28:07.246 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 14:30:03.999 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 14:31:06.553 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 15:28:04.580 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 15:30:02.989 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 15:31:05.253 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 16:28:03.917 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 16:30:03.220 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 16:31:05.483 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 17:28:04.291 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 17:30:04.167 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 17:31:06.433 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 18:28:04.604 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 18:30:03.462 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 18:31:05.730 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 19:28:04.318 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 19:30:03.570 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 19:31:05.833 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 20:28:05.979 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 20:30:02.686 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 20:31:04.950 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 21:28:06.001 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 21:30:13.589 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2017/01/22 21:30:28.201  10.0275,  3.35367,    0.845, 4130, 125, 79, 22 Jan 2017 21:30:21
2017/01/22 21:31:04.014 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 22:28:04.573 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 22:30:02.101 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2017/01/22 22:31:04.364 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2017/01/22 23:28:05.196 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2017/01/22 23:30:12.984 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2017/01/22 23:30:27.626   9.9874,  3.31798,    0.834, 4124, 127, 80, 22 Jan 2017 23:30:21
2017/01/22 23:31:02.820 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
