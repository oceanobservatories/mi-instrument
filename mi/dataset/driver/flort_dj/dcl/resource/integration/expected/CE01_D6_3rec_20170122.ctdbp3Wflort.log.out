
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE01ISSM | SBD17 | 06-FLORTD000 | 3694032000 | telemetered | 3694048226.8059997559 |          6 |         3694048220 | 3694048226.8059997559 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            3694 |              80 |            121 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3694032000 | telemetered | 3694109428.2010002136 |          6 |         3694109421 | 3694109428.2010002136 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            4130 |              79 |            125 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3694032000 | telemetered | 3694116627.6259999275 |          6 |         3694116621 | 3694116627.6259999275 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            4124 |              80 |            127 |        null

(3 rows)
